src/zenml/model/model.py

```diff
@@ -503,9 +503,25 @@ def _root_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         values["suppress_class_validation_warnings"] = True
         return values
 
-    def _validate_config_in_runtime(self) -> None:
-        """Validate that config doesn't conflict with runtime environment."""
-        self._get_or_create_model_version()
+    def _validate_config_in_runtime(self) -> bool:
+        """Validate that config doesn't conflict with runtime environment.
+
+        Returns:
+            True, if the server is part of ZenML Cloud, False otherwise.
+        """
+        from zenml.utils.dashboard_utils import get_model_version_url
+
+        model_version = self._get_or_create_model_version()
+
+        model_version_url = get_model_version_url(model_version.id)
+        if model_version_url:
+            logger.info(
+                f"Dashboard URL for Model Version with name {model_version.name} "
+                f": {model_version_url}"
+            )
+            return True
```

Comment:
I put the logic in there to avoid having to call `_get_or_create_model_version()` twice, as the validate function already makes that call. Maybe I can return the output of that fn (a model version) from the validation function and then call a utiils function to make the cloud server check?

Reply:
Yep that sounds good!